roles/internal/righttoknow/tasks/main.yml

```diff
@@ -353,7 +353,7 @@
     - staging
   notify: nginx reload
 
-- name: Copy across the main Alaveteli config (production)
+- name: Apply Alaveteli config (production)
```

Comment:
Is there any particular reason we are renaming these tasks? It's not clear the benefit of doing this.

Reply:
It sounds to me like a much clearer description of what it's doing here.